holmes/plugins/toolsets/confluence/confluence.py

```
@@ -0,0 +1,376 @@
+            return StructuredToolResult(
+                status=StructuredToolResultStatus.SUCCESS,
+                data=data,
+                params=params,
+            )
+        except requests.exceptions.HTTPError as e:
+            error_detail = f"HTTP {e.response.status_code}"
+            try:
+                error_body = e.response.json()
+                if "message" in error_body:
+                    error_detail = f"{error_detail}: {error_body['message']}"
+            except Exception:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)